src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,11 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
     public String getRandomColor() {
-        return null;
+        int randomColorIndex = new Random().nextInt(Color.values().length);
```

Comment:
Random should be class-level variable
[
Think about which variables should be local in the method and which should be class-level](https://github.com/mate-academy/jv-lottery/blob/master/checklist.md#think-about-which-variables-should-be-local-in-the-method-and-which-should-be-class-level)